nonprofit-board-management.php

```diff
@@ -175,6 +201,32 @@ public function add_board_roles(){
       }
     }
 
+	/**
+	 * Remove the Board Member user role on plugin deactivation.
+	 *
+	 * @param boolean $network_wide If the plugin is network activated.
+	 */
+	public function deactivate_plugin( $network_wide ) {
+
+		/**
+		 * If the plugin is network activated we need to remove the
+		 * Board Member role for each subsite.
+		 */
+		if ( is_multisite() && $network_wide ) {
+
+			global $wpdb;
+			$blog_ids = $wpdb->get_col( "SELECT blog_id FROM $wpdb->blogs" );
+			foreach ( $blog_ids as $blog_id ) {
+				switch_to_blog( $blog_id );
+				$this->remove_board_roles();
+				restore_current_blog();
+			}
+
+		} else {
+
+			$this->remove_board_roles();
+		}
+	}
 
     /*
      * Remove the board member role when the plugin is deactivated.
```

Comment:
Do you want to update this comment since it still references deactivation? This function has been separated out from deactivation so I figure it should be modified.

Reply:
Yeah I can do that. This function is only called on plugin deactivation but I'll try to clarify.